New change alert monitor guide (#19114)
DataDog/documentation: content/en/monitors/guide/change-alert.md

| 5 minutes        | The evaluation window. For more information, see the [Monitor Configuration][1] documentation.  | This can be N minutes, hours, days, weeks, or at most one month. |
| 1 hour           | The timeframe that you wish to shift the query by.                                              | This can be N minutes, hours, days, weeks, or at most one month ago.|

### Change and change %

There are two options when configuring a change alert detection, **Change** and **% Change**. 

This determines the way the monitor evaluates as expressed in the formula section in the following table:

| Option   | Description                                                        | Formula              |
| -------  | ------------------------------------------------------------------ | -------------------- |
| Change   | The absolute change of the value.                                  | `a - b`              |
| % Change | The percentage change of the value compared to its previous value. | `((a - b) / b) * 100`|

In both cases, `Change`, and `% Change` can be either positive or negative. 

## Troubleshooting a change alert evaluation

> Change alerts can be mistaken to be false positives. You can reconstruct the metric queries with Notebooks to verify the results of your monitor evaluation. 

Reviewer: ```suggestion
If you suspect that your change alert evaluation is a false positive, you can reconstruct the metric queries with a Notebook to verify the results. 
```
